palantir-java-format/src/main/java/com/palantir/javaformat/java/CommandLineOptions.java

```diff
@@ -45,6 +45,7 @@ final class CommandLineOptions {
     private final Optional<String> assumeFilename;
     private final boolean reflowLongStrings;
     private final boolean outputReplacements;
+    private final Optional<Long> warnOnExpensiveFileDurationMillis;
```

Comment:
I would probably long term want to be able to set a `failIfDurationExceeded` option as well. Ideas for if this flag should be altered or if it's just another config?

Reply:
just another config, imo